pkg/package-server/apis/operators/packagemanifest_types.go

```diff
@@ -81,6 +80,17 @@ type PackageChannel struct {
 
 	// CurrentCSVSpec holds the spec of the current CSV
 	CurrentCSVDesc CSVDescription
+
+	// Entries lists all CSVs in the channel, with their upgrade edges.
+	Entries []ChannelEntry
```

Comment:
nit: would refactoring to `Entries` to `ChannelEntries` improve readability?

Reply:
I could go either way. We're already in the `PackageChannel` struct, and none of the other fields have a `Channel` prefix, so my instinct was just `Entries`.

I _did_ originally have the struct type `ChannelEntry` called `Entry`. I changed from `Entry` to `ChannelEntry` in that case because the type is defined at the package level, and it made sense to me for usage of the struct from outside the package to look like `operators.ChannelEntry` rather than `operators.Entry`